src/schemas/Program/index.js

```diff
@@ -284,6 +285,7 @@ const programServicePrivateFields = [
 	'genomicDonors',
 	'membershipType',
 	'users',
+	'dataCenter',
 ];
```

Comment:
add dataCenter to private fields. (when users query items in this list, system will use the private resolver to prevent people who don't have access getting information )